index.html

```diff
@@ -137,14 +137,10 @@ <h3>
             `TCPSocketOptions` dictionary
           </h3>
           <pre class="idl">
-          dictionary TCPKeepAliveOptions {
-            boolean enable;
-            [Clamp] unsigned short delay; 
-          };
-          
           dictionary TCPSocketOptions : SocketOptions {
             boolean noDelay;
-            TCPKeepAliveOptions keepAliveOptions;
+            boolean keepAlive;
+            [Clamp] unsigned long keepAliveDelay;
```

Comment:
Note the advice in https://w3ctag.github.io/design-principles/#numeric-types; I would suggest `[EnforceRange] unsigned long long`.

Reply:
`[EnforceRange]` definitely makes sense (it's actually what I was looking for!), but is `long long` really necessary? `keepAliveDelay` value will be limited by ~ `const * 2^27` (that's one day in millis).

Reply:
One more point is that the network signature of `SetKeepAlive` accepts an integer, so supplying it with an `int64` may cause some weird behaviour:
```
bool SetKeepAlive(bool enable, int delay_secs);
```

Reply:
If it's beyond the underlying platform capabilities to accept large integers, then I guess that's reasonable. It feel arbitrary that 2^32 - 1 is OK but 2^32 is not; in particular it's arbitrary in a way that we like to avoid in web APIs, per the above link. But oh well.

Reply:
I see your point! Wish we could allow values up to 2^35, but it doesn't work this way.
I've uploaded a patch, please lgtm :)